qemu/tests/snp_multi_vm.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+from virttest import data_dir as virttest_data_dir
+from virttest import env_process, error_context
+from virttest.utils_misc import get_mem_info, normalize_data_size, verify_dmesg
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu snp basic test on Milan and above host:
+    1. Check host snp capability
+    2. Boot snp VM
+    3. Verify snp enabled in guest
+    4. Check snp qmp cmd and policy
```

Comment:
Please update the doc string, here should be snp multi vm related instead of snp basic config test. If I was wrong, please correct me. Thanks.